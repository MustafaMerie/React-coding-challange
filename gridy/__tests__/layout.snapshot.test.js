import Layout from '../components/Layout'
import renderer from 'react-test-renderer'

it('Layout renders correctly', () => {
  const tree = renderer.create(<Layout />).toJSON()
  expect(tree).toMatchSnapshot()
})