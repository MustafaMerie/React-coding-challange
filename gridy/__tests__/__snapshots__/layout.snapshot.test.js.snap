// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
Array [
  <div
    className="header"
  >
    <div
      className="overlay"
    />
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="header_text col"
        >
          <h1
            className="display-3"
          >
            Free Stock Photos
          </h1>
          <p
            className="lead"
          >
            The internet's source of freely-usable images.
            <br />
            Powered by creators everywhere.
          </p>
        </div>
      </div>
    </div>
  </div>,
  <main />,
]
`;
